rfcs/20220815-v3.0-API-cleanup/README.md

```diff
@@ -0,0 +1,91 @@
+        * End user must update `batch_normalization_forward` and `batch_normalization_backward` classes with proper calls to primitive descriptors.
+    * [`shuffle_forward`](dnnl.hpp#L11687) and [`shuffle_backward`](dnnl.hpp#L11769) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `shuffle_forward` and `shuffle_backward` classes with proper calls to primitive descriptors.
+    * [`eltwise_forward`](dnnl.hpp#L6656) and [`eltwise_backward`](dnnl.hpp#L6757) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `eltwise_forward` and `eltwise_backward` classes with proper calls to primitive descriptors.
+    * [`prelu_forward`](dnnl.hpp#L12647) and [`prelu_backward`](dnnl.hpp#L12742) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `prelu_forward` and `prelu_backward` classes with proper calls to primitive descriptors.
+    * [`lrn_forward`](dnnl.hpp#L6165) and [`lrn_backward`](dnnl.hpp#L6268) constructors relying on a single data descriptor instead of source and destination will be replaced with constructors providing separate source and destination memory descriptors.
+        * End user must update `lrn_forward` and `lrn_backward` classes with proper calls to primitive descriptors.
+
+* Primitive descriptors:
+    * C API [`dnnl_convolution_forward_desc_init`](dnnl.h#1512), [`dnnl_convolution_backward_data_desc_init`](dnnl.h#1589), and [`dnnl_convolution_backward_weights_desc_init`](dnnl.h#1663) calls will be removed. [`dnnl_dilated_convolution_forward_desc_init`](dnnl.h#1555), [`dnnl_dilated_convolution_backward_data_desc_init`](dnnl.h#1626), and [`dnnl_dilated_convolution_backward_weights_desc_init`](dnnl.h#1704) will be renamed to `dnnl_convolution_forward_desc_init`, `dnnl_convolution_backward_data_desc_init`, and `dnnl_convolution_backward_weights_desc_init` correspondently. Note that these API calls will be updated to have three more arguments according to [this RFC](https://github.com/oneapi-src/oneDNN/pull/1391).
+        * End user must update calls that will be removed with dilation parameter or change the name of modified calls.
+    * Sum [`primitive_desc`](dnnl.hpp#4519) constructors, Concat [`primitive_desc`](dnnl.hpp#4422) constructors, and Shuffle [`primitive_desc`](dnnl.hpp#11727) constructor will change a signature, where attributes and engine will be swapped, to align with rest primitive_desc constructors from other classes.
```

Comment:
I guess this goes in C/C++ types section.

Reply:
It's under "Primitive descriptors" section. Would you like to move whole section under C/C++ Types section?